runtime/lib/component/lifetime.js

```diff
@@ -596,6 +596,9 @@ LaVieEnPile.prototype.onEvict = function onEvict (appId) {
   }
   process.nextTick(() => {
     this.emit('evict', appId)
+    if (!this.getCurrentAppId()) {
+      this.emit('idle')
+    }
```

Comment:
@legendecas could this be present the lifetime is idle?

Reply:
> @legendecas could this be present the lifetime is idle?

I will check the event `evict` again

Reply:
idle state shall be checked and memo-ed before `process.nextTick`